generics/services/project.js

```diff
@@ -0,0 +1,63 @@
+/**
+ * name : project.js
+ * author : Mallanagouda R Biradar
+ * Date : 11-Apr-2025
+ * Description : Projcet service related information.
+ */
+
+const request = require('request');
+const projectServiceUrl = process.env.INTERFACE_SERVICE_URL;
+
+// Function to fetch the project template based on the given externalId
+const listById = function (userToken, externalId) {
+  return new Promise(async (resolve, reject) => {
+    try {
+      // Construct the URL for the project service
+      let url = `${projectServiceUrl}${process.env.PROJECT_SERVICE_NAME}${messageConstants.endpoints.PROJECT_LIST_BY_ID}`;
+        
+      // Set the options for the HTTP GET request
+      const options = {
+        headers: {
+          'content-type': 'application/json',
+          'X-auth-token': userToken,
+        },
+        json: {
+          'externalIds': externalId,
+        },
+      };
+      
+      request.post(url, options, userReadCallback);
+      let result = {
+        success: true,
+      };
+      // Handle callback fucntion
+      function userReadCallback(err, data) {
```

Comment:
@MallanagoudaB why this function called userRead callback? this call is to project service rename this